openmls/src/framing/mod.rs

```diff
@@ -200,14 +200,14 @@ where
     ///
     /// Note: This function makes sure that the input is fully consumed, i.e., that there is no
     ///       trailing data.
-    fn try_from_bytes(bytes: impl AsRef<[u8]>) -> Result<Self, TlsFromBytesError>;
+    fn tls_deserialize_complete(bytes: impl AsRef<[u8]>) -> Result<Self, TlsFromBytesError>;
```

Comment:
Why the rename? I don't think `tls_deserialize_complete` is what we want. Staying in the Rust terminology we could call it `_exact`.